### PR TITLE
Some Gluth fixes...

### LINKED ISSUE #736: Gluth ignore tank
Description:

There is a problem with the boss in Naxxramas gluth.

Current behaviour: Tell us what happens

When there are zombie boss runs to them ignoring the agro tank

Expected behaviour: Tell us what should happen instead

After the appearance of the zombie, the gluth should react to the agro tank if there is

Steps to reproduce the problem:

1. Go to Naxxramas
2. Start killing the gluth
3. Waiting for the appearance of zombies

Branch(es): 0.x / 1.x / master (Specify the branch(es) affected by this issue) 

Last revision b527073, master

AC hash/commit:  

Operating system:  Linux, Ubuntu 16.04

Modules:  Anticheat, Transmogrification, Premium account

Other Customizations: Made 2 vendors

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/52572564-gluth-ignore-tank?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by FALL1N1**
Changes:
- Fixed infinite stacks (and eventually crash) of infected wound. (Also fixes the stranglethorn vale zombies' DoT stack issue)
- Fixed zombie spawn location (3 gates, only 1 used in 10man, all 3 in 25)
- Fixes zombie chow distance, closes #736

### CODE REVIEW

File: src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp [hunk 1]
```diff
@@ -174,15 +174,20 @@ class boss_gluth : public CreatureScript
...
-                        me->SummonCreature(NPC_ZOMBIE_CHOW, zombiePos[urand(0,2)]);
-                        (rand == 2 ? rand = 0 : rand++);
-                    }
-
-                    events.RepeatEvent(10000);
-                    break;
+                        uint8 rand = urand(0,2);
+                        for (int32 i = 0; i < RAID_MODE(1,2); ++i)
+                        {
+                            // 10n - should spawn only from mid gate 25 from all 3
```
Comment (Viste) [inline src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp]: not very clear comment, or take it away, or write in detail

File: src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp [hunk 2]
```diff
@@ -174,15 +174,20 @@ class boss_gluth : public CreatureScript
...
+                            // \1 |0 /2 pos
```
Comment (Viste) [inline src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp]: and here

**Comment by Dreaxxx** [comment]
Distance for eating zombie is fine , but in 10 man , zombies appear 1 by one but not only in the mid gate , they continue to spawn at the 3 gates.

**Comment by FALL1N1** [comment]
That comment is for gate naming, e.g 1st is left, 0 is mid, 2 is right, it should stay as it is.

**Comment by BarbzYHOOL** [comment]
Personally I don't understand the comment either at first sight. it looks like a secret code :P

Also why "10n"? what does "n" mean? why not "p" for players ? lol

Btw Fall1n, your PR makes this PR useless, right? https://github.com/azerothcore/azerothcore-wotlk/pull/830

**Comment by FALL1N1** [comment]
@BarbzYHOOL should've been 10man, yes. Habbit of mine to call Naxxramas by difficulty since it's only normal mode, not heroic, and yes, it closes #830 and makes the pr useless since this is a proper way to handle it.

**Comment by BarbzYHOOL** [comment]
I tried to detail the comment as much as I can without having doing any raid in WOTLK so please confirm that I wrote the right thing :P

**Comment by BarbzYHOOL** [comment]
Ok, lol I just realize Dreaxxx had given his opinion on the comment and I had not read it... -_-'

Anyway, thank you @FALL1N1 for that nice PR!

Thank you @Viste and @Dreaxxx for feedback!

**Pull request merged**